### DISCUSSION TITLE
Buddhist Studies Time Authority Database

### DISCUSSION THREAD

**Discussion by atomrab** [periodo/periodo-data#92] [New period authorities]
Christine Fernsebner-Eslao pointed this out after our LD4 paper (or rather her partner, who specializes in Buddhist studies, did): http://authority.dila.edu.tw/time/. I have a vague memory (@rybesh, can you confirm?) that we'd talked about incorporating definitions from Dharma Drum a long time ago, but we never got there. Should we come back around to this? I can't read Chinese, so this isn't something I can assess.

**Comment by rybesh** [comment]
Yes, the Dharma Drum period authority predates PeriodO, and is actually the first true linked data period gazetteer I'm aware of, so it would be good to include.
